### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of null

### ISSUE THREAD

**Issue by Lachney** [webbukkit/dynmap#3101]
```
jquery-1.11.0.js?_=3.1-SNAPSHOT-408:358 Uncaught TypeError: Cannot read property 'length' of null
    at Function.each (jquery-1.11.0.js?_=3.1-SNAPSHOT-408:358)
    at DynMap.configure (map.js?_=3.1-SNAPSHOT-408:85)
    at map.js?_=3.1-SNAPSHOT-408:37
    at success (jquery.json.js?_=3.1-SNAPSHOT-408:82)
    at Object.obj.success (jquery.json.js?_=3.1-SNAPSHOT-408:19)
    at fire (jquery-1.11.0.js?_=3.1-SNAPSHOT-408:3099)
    at Object.fireWith [as resolveWith] (jquery-1.11.0.js?_=3.1-SNAPSHOT-408:3211)
    at done (jquery-1.11.0.js?_=3.1-SNAPSHOT-408:9310)
    at XMLHttpRequest.callback (jquery-1.11.0.js?_=3.1-SNAPSHOT-408:9720)
```

**Issue Description:** I'm trying to setup an external web server using nginx on Ubuntu 20.04. When accessing the map on the browser, it only shows up a black screen. In the console, it outputs the error above. I have checked my folder permissions, seemed all good. I have disabled the internal web server, I am using MySQL for storage and have enabled the "JsonFileClientUpdateComponent" and commented out InternalClientUpdateComponent.

* **Dynmap Version:** 3.1-beta-3-406
* **Server Version:** git-Paper-134 (Minecraft 1.16.1)
* **Pastebin of Configuration.txt:** *https://pastebin.com/4Ts36nWA*
* **Server Host (if applicable):** Selfhosted
* **Pastebin of crashlogs or other relevant logs:** no errors caused by plugin
* **Other Relevant Data/Screenshots:** I'm using Ubuntu 20.04, PHP 7.4, NGINX 1.18.0
* **Steps to Replicate:** 
  * Disable internal webserver then follow the guide listed here: https://github.com/webbukkit/dynmap/wiki/External-Webserver-Basics

 [X] *I have looked at all other issues and this is not a duplicate*  
 [X] *I have been able to replicate this*

**Comment by FedUpWith-Tech** [comment]
Related to #3075

**Comment by FedUpWith-Tech** [comment]
FYI: This issue is as far as I am aware limited to nginx and mysql, apache and other web hosts haven't had this issue as well as other storage options. I know this isn't a great solution and I'm investigating why that specific combo is causing so many issues but there are workarounds available

**Comment by iperrealistico** [comment]
And here I am too! Found ya.

Had the brilliant idea to setup an NGINX server for my old Dynmap.
It took me an entire day but at the end I got it working, I had to configure NGINX to use PHP, also.
For an hour or two I was getting "could not retrieve configuration: error" then once i got PHP set up, it disappeared.

All I get now is a black screen and this error in console:
![image](https://user-images.githubusercontent.com/25549272/97614270-06c55680-1a1a-11eb-8e71-9883aacde462.png)

www.peacefulvanilla.club/map.html

I am kinda happy to have found this issue, but also annoyed 😢
